pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -148,6 +149,27 @@ func (brokerStatusRESTStrategy) PrepareForUpdate(ctx genericapirequest.Context,
 	}
 	// status changes are not allowed to update spec
 	newBroker.Spec = oldBroker.Spec
+
+	foundReadyConditionTrue := false
+	for _, condition := range newBroker.Status.Conditions {
+		if condition.Type == sc.BrokerConditionReady && condition.Status == sc.ConditionTrue {
+			foundReadyConditionTrue = true
+			break
+		}
+	}
+	if foundReadyConditionTrue {
+		glog.Infof("Found true ready condition for Broker %v/%v; updating checksum", newBroker.Namespace, newBroker.Name)
+		// This status update has a true ready condition; update the checksum if necessary
```

Comment:
the language is probably copied from another resource, but yes, we should fix it.